index.ts

```diff
@@ -134,7 +134,9 @@ if (!oauthEnabled) {
 
 app.get("/qr/:type/:room?", async (req: express.Request, res: express.Response) => {
   const options: QRCode.QRCodeToBufferOptions = {
+    errorCorrectionLevel: "L",
     width: 300,
+    margin: 0,
```

Comment:
without a margin, i.e. without a light-color border, QR codes may be come un-readable when copied to a dark-color background